src/components/ha-top-app-bar-fixed.ts

```diff
@@ -27,6 +27,9 @@ export class HaTopAppBarFixed extends TopAppBarFixedBase {
         padding-inline-start: 20px;
         padding-inline-end: initial;
       }
+      .header {
```

Comment:
I don't think there is an element that has the `header` class? Have you tested this?

Reply:
I think this is where this should go:
https://github.com/home-assistant/frontend/blob/35cf3063cb09e583c607239cd3658621a2e04312/src/panels/lovelace/hui-root.ts#L904

Reply:
> I don't think there is an element that has the `header` class? Have you tested this?

Oh sorry, I got confused here, if I want to add it directly to ha-top-app-bar-fixed can I just remove ".header"?

Reply:
<img width="305" alt="Screenshot 2024-04-09 at 15 13 14" src="https://github.com/home-assistant/frontend/assets/5808343/15a3c9dd-9933-4b3e-8637-92a02fd215d9">
I think I am adding it twice

Reply:
@Nezz but would it then go to all headers in HA?

Reply:
Oh right, that way it'd only get added to the dashboards. Add it under this one:
https://github.com/home-assistant/frontend/blob/38f8b4e0877da7ca8e19e2650424ea016f096480/src/components/ha-top-app-bar-fixed.ts#L18
and this one:
https://github.com/home-assistant/frontend/blob/38f8b4e0877da7ca8e19e2650424ea016f096480/src/components/ha-top-app-bar.ts#L18